src/main/java/com/alkemy/ong/controller/UserController.java

```diff
@@ -50,6 +55,7 @@ public UserDto userMe(@RequestHeader("Authorization") String jwt){
     }
 
```

Comment:
we could take advantage of this pr to add the error message with paramNotFound on de catch

   } catch (Exception e) {
            throw new ParamNotFound("There is no Member with the entered Id");
        }

Reply:
you are right I will add the errors

Reply:
Approved